cli/crates/server/src/bridge/types.rs

```diff
@@ -57,6 +65,8 @@ pub struct Record {
     pub entity_type: String,
     pub relation_names: Value,
     pub document: Value,
-    pub created_at: String,
-    pub updated_at: String,
+    #[serde(serialize_with = "serialize_dt_to_rfc3339")]
+    pub created_at: DateTime<Utc>,
```

Comment:
@cnixon-gb Is this already done in the worker side? We originally had it like this but removed it both due to the warning in `chrono` and due to it basically adding an unneeded operation

Reply:
Minus the tag of course